ecs-cli/modules/config/params.go

```diff
@@ -108,6 +109,15 @@ func NewCLIParams(context *cli.Context, rdwr ReadWriter) (*CLIParams, error) {
 		ecsConfig.CFNStackName = flags.CFNStackNamePrefixDefaultValue + ecsConfig.Cluster
 	}
 
+	if ecsConfig.Cluster == "" {
```

Comment:
To be clear, it will only log a warning if cluster isn't specified anywhere -- including as a flag. 

If this check should be in a different place, let me know!

Reply:
So the issue with this is that it will log this warning for all commands, even the ECR commands, which do not need or have any use for a cluster name.

Reply:
I'll move it over to 'ecs-cli ps'. I'll do a quick scan to make sure other codes work the same way.